go.mod

```diff
@@ -3,10 +3,9 @@ module github.com/Microsoft/opengcs
 go 1.12
 
 require (
-	github.com/Microsoft/go-winio v0.4.12
-	github.com/Microsoft/hcsshim v0.0.0-20190404205635-cf1c2137fae1
-	github.com/containerd/containerd v1.3.0
-	github.com/containerd/continuity v0.0.0-20190815185530-f2a389ac0a02 // indirect
+	github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5
+	github.com/Microsoft/hcsshim v0.8.7-0.20191007171158-cf76d8132ece
+	github.com/containerd/containerd v1.3.0-beta.2.0.20190828155532-0293cbd26c69
```

Comment:
Sorry this one is older than v1.3.0. How are you doing this it seems odd the changes its making here.

Reply:
By running go get github.com/Microsoft/hcsshim@master.

Reply:
By running go get github.com/Microsoft/hcsshim@master.